### PR TITLE
Moved adventure confetti emojis to yaml 

### PR DESCRIPTION
**Pull request by fRedelaar**
**Description**

For issue #1414 the confetti emojis have been moved from app.ts to the English yaml file. This is so content developers can adjust emojis with more ease in the future.

### CODE REVIEW

File: coursedata/texts/en.yaml [hunk 1]
```diff
@@ -52,6 +52,13 @@ ui:
     not_enrolled: "Looks like you are not in this class!"
     retrieve_class: "Only teachers can retrieve classes"
 ClientErrorMessages:
+    Emoji_turtle: "🐢"
```
Comment (Felienne) [inline coursedata/texts/en.yaml:55]: Great you are working on this @fRedelaar! However, maybe we can make a dictionary here, that would look a bit nicer? Something like:

Client
```suggestion
    ClientErrorMessages:
        turtle: "🐢",
        rock: "✂️, 📜, 🗻"
```

Then in ts we can look up a list with the name of the adventure, simplifying the code too? (see second suggestion)

File: static/js/app.ts [hunk 2]
```diff
@@ -874,15 +874,26 @@ function getConfettiForAdventure(adventure: string){
 
   switch (adventure) {
     case 'turtle':
-      return [['🐢']];
+      let turtleEmoji = Array.from(ErrorMessages['Emoji_turtle'])
```
Comment (Felienne) [inline static/js/app.ts:877]: With the dict in yaml, we can do something like `ErrorMessages[adventure]` here, reducing the neem for the switch.

**Review by Felienne — CHANGES_REQUESTED** [review]

File: static/js/app.ts (hunk 2 above)
Comment (fRedelaar) [reply static/js/app.ts]: Great suggestion! I've pushed a new commit.

**Review by Felienne — APPROVED** [review]
Awesome, thanks!!

**Pull request merged**